From 6a398ab4a5a60b480833b2e2664110aaf1c7bacf Mon Sep 17 00:00:00 2001
From: Harini Katakam <harini.katakam@xilinx.com>
Date: Thu, 24 Sep 2020 13:16:55 +0530
Subject: [PATCH 1617/1851] net: xilinx: Add missing kernel-doc descriptions

commit 0ef3948b6cb4f2a9c9587cda1bac404abc4743f5 from
https://github.com/Xilinx/linux-xlnx.git

Add missing kernel-doc descriptions for clk, tx_bd_num and
rx_bd_num.

Signed-off-by: Harini Katakam <harini.katakam@xilinx.com>
Reviewed-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/net/ethernet/xilinx/xilinx_axienet.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/net/ethernet/xilinx/xilinx_axienet.h b/drivers/net/ethernet/xilinx/xilinx_axienet.h
index ba6696ba66cc..fa6e22425aee 100644
--- a/drivers/net/ethernet/xilinx/xilinx_axienet.h
+++ b/drivers/net/ethernet/xilinx/xilinx_axienet.h
@@ -684,6 +684,7 @@ enum axienet_tsn_ioctl {
  * @ndev:	Pointer for net_device to which it will be attached.
  * @dev:	Pointer to device structure
  * @phy_node:	Pointer to device node structure
+ * @clk:	AXI bus clock
  * @mii_bus:	Pointer to MII bus structure
  * @regs_start: Resource start for axienet device addresses
  * @regs:	Base address for the axienet_local device address space
@@ -712,6 +713,8 @@ enum axienet_tsn_ioctl {
  * @options:	AxiEthernet option word
  * @last_link:	Phy link state in which the PHY was negotiated earlier
  * @features:	Stores the extended features supported by the axienet hw
+ * @tx_bd_num:	Number of TX buffer descriptors.
+ * @rx_bd_num:	Number of RX buffer descriptors.
  * @max_frm_size: Stores the maximum size of the frame that can be that
  *		  Txed/Rxed in the existing hardware. If jumbo option is
  *		  supported, the maximum frame size would be 9k. Else it is
-- 
2.31.1

